mappings/net/minecraft/server/command/JfrCommand.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/unmapped/C_vkmlkrvv net/minecraft/server/command/JfrCommand
```

Comment:
Should jfr be `Jfr` or `JFR`? as it stands for `Java Flight Recorder`, I would lean to `JFR`.

Reply:
[Current conventions](https://github.com/QuiltMC/quilt-mappings/blob/21w37a/CONVENTIONS.md#abbreviations) state that it should be Jfr